t/25-cache-service.t

```diff
@@ -0,0 +1,454 @@
+#! /usr/bin/perl
+
+# Copyright (c) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+BEGIN {
+    unshift @INC, 'lib', 't/lib';
+    use Mojo::File qw(path tempdir);
+    use FindBin;
+    use File::Path qw(remove_tree make_path);
+
+    my $basedir = path(tempdir, 't', 'cache.d');
+    remove_tree($basedir);
+    $ENV{CACHE_DIR}      = path($basedir, 'cache');
+    $ENV{OPENQA_BASEDIR} = $basedir;
+    $ENV{OPENQA_CONFIG}  = path($basedir, 'config')->make_path;
+    path($ENV{OPENQA_CONFIG})->child("workers.ini")->spurt("
+[global]
+CACHEDIRECTORY = $cachedir
+CACHELIMIT = 100");
+}
+
+use strict;
+use warnings;
+# Avoid warning error: Name "Config::IniFiles::t/cache.d/cache/config/workers.ini" used only once
```

Comment:
It is not obvious how the comment relates to the subsequent line. Or does it relate to the line above?